moveit_ros/planning/plan_execution/src/plan_execution.cpp

```diff
@@ -370,7 +370,9 @@ moveit_msgs::MoveItErrorCodes plan_execution::PlanExecution::executeAndMonitor(E
     // convert to message, pass along
     moveit_msgs::RobotTrajectory msg;
     plan.plan_components_[i].trajectory_->getRobotTrajectoryMsg(msg);
-    if (!trajectory_execution_manager_->push(msg))
+
+    // trajectory execution manager handles empty controller name vectors in its configure(), so we just pass them along
```

Comment:
The comment isn't very helpful in the long run. Either drop it or replace it by a more generic one.